src/Angor/Avalonia/AngorApp.Model/BitcoinAddressValidator.cs

```diff
@@ -0,0 +1,269 @@
+namespace AngorApp.Model;
+
+using System.Numerics;
+using System.Security.Cryptography;
+
+public class BitcoinAddressValidator
```

Comment:
I don't mind bringing code here but NBitcoin has such a validator no? but it is ok

Reply:
Ah, yes! I created this helper using AI just to add some quick validation. It think it would be good to switch to existing NBitcoin code later on, for example, as soon as we start using the Shared project. What do you think?